strawberry_django/receivers.py

```diff
@@ -0,0 +1,17 @@
+import contextlib.suppress
```

Comment:
Imoporting `contextlib` is enough. It will also solve the typing issue being raised

```suggestion
import contextlib
```